tests/resources/promise-rejection-events.js

```diff
@@ -642,12 +844,12 @@ function onUnhandledSucceed(t, expectedReason, expectedPromiseGetter) {
 function onUnhandledFail(t, expectedPromiseGetter) {
   var unhandled = function(evt) {
     if (evt.promise === expectedPromiseGetter()) {
-      t.unreached_func('unhandledrejection event is not supposed to be triggered');
+      t.assert_unreached('unhandledrejection event is not supposed to be triggered');
```

Comment:
Argh, I thought I tested this. The following should work in theory:

```js
t.step(function() {
  assert_unreached('...');
});
t.done(); // maybe this line is optional
```

if not I can do better when I get in to work in a few hours; just woke up.